plugin/teksi_wastewater/interlis/interlis_model_mapping/interlis_importer_to_intermediate_schema.py

```diff
@@ -445,6 +445,10 @@ def wastewater_structure_common(self, row):
             ),
             "contract_section": row.baulos,
             "detail_geometry3d_geometry": ST_Force3D(row.detailgeometrie),
+            # TODO : NOT MAPPED VSA-DSS 3D
```

Comment:
@sjib could you raise an issue for that and mention the requirements to make it 1:1 with VSA DSS 2020 3D?

Reply:
There exists INTERLIS VSA-DSS 2020 3D - but it is not used currently as we have a "problem" in INTERLIS that we can create e.g. detail_geometry3d, but will not have 3D information in most cases for a whole community right now. And if we do not have that, then the validation of INTERLIS VSA-DSS 2020 3D fails. But it is also a little bit a henn - egg problem. I will raise a discussion as proposed.

Reply:
https://github.com/orgs/teksi/discussions/109